web/src/main/webapp/my-app/marketplace/controllers.js

```diff
@@ -57,10 +57,10 @@ define(['angular', 'jquery', 'require'], function(angular, $, require) {
           && portlet.portletName // there is a portletName
           && portlet.portletName.indexOf('cms') != -1; // static content portlet
       };
-
+      $scope.GuestMode = $sessionStorage.GuestMode;
```

Comment:
This might not be what we want. I think instead we'd like it to get `GuestMode` from the service, rather than sessionStorage. At the moment, I forget what that looks like in other places. I'll take a look this afternoon

Reply:
Yeah, we shouldn't replace this with a sessionStorage lookup, that's going to be removed as well.  You should be using the `mainService.isGuest()` async function. Here's an example: https://github.com/uPortal-Project/uportal-app-framework/pull/680/files#diff-5dc1012888a1a7946853bbacd81416b6R134